modules/common-graphql/src/main/scala/observe/common/ObsQueriesGQL.scala

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.common
+
+import clue.GraphQLOperation
+import clue.annotation.GraphQL
+import observe.schemas.ObservationDB
+// gql: import io.circe.refined._
+
+object ObsQueriesGQL {
+
+  @GraphQL
+  trait ProgramObservationsQuery extends GraphQLOperation[ObservationDB] {
+    val document = """
+      query($first: Int = 2147483647) {
+        constraintSets(programId: "p-2", first: $first) {
+          nodes {
+            id
+            name
+          }
+        }
+
+        observations(programId: "p-2", first: $first) {
+          nodes {
+            id
+            observationTarget {
+              type: __typename
+              ... on Target {
+                targetId: id
+                targetName: name
+              }
+              ... on Asterism {
+                asterismId: id
+                asterismName: name
+              }
+            }
+            constraintSet {
+              id
+              imageQuality
+              cloudExtinction
+              skyBackground
+              waterVapor
+            }
+          }
+        }
+      }
+    """
+
+  }
+
+  @GraphQL
+  trait ProgramObservationsEditSubscription extends GraphQLOperation[ObservationDB] {
+    val document = """
+      subscription {
+        observationEdit(programId:"p-2") {
+          id
+        }
+      }
+    """
+  }
+
+  @GraphQL
+  trait ProgramCreateObservation extends GraphQLOperation[ObservationDB] {
```

Comment:
Do you need to create/delete observations in `observe`?

Reply:
No